server/src/DBTablesMonitoring.cc

```diff
@@ -782,6 +784,103 @@ static const DBAgent::TableProfile tableProfileIncidents =
 			    NUM_IDX_INCIDENTS,
 			    indexDefsIncidents);
 
+// ----------------------------------------------------------------------------
+// Table: incident_status_histories
+// ----------------------------------------------------------------------------
+static const ColumnDef COLUMN_DEF_INCIDENT_STATUS_HISTORIES[] = {
+{
+	"id",                              // columnName
+	SQL_COLUMN_TYPE_INT,               // type
```

Comment:
It should be BIGUINT because the number of rows of incident_status_history table will be larger than events.
(When an event is registered, an indent & an incident_status_history will be also created by default.
Then a user changes the status of the incident, another incident_status_history for it will be created.)